include/oneapi/dpl/internal/distributed_ranges_impl/shp/remote_vector.hpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include <oneapi/dpl/internal/distributed_ranges_impl/shp/allocators.hpp>
+#include <oneapi/dpl/internal/distributed_ranges_impl/shp/vector.hpp>
+
+namespace oneapi::dpl::experimental::dr::shp
+{
+
```

Comment:
I think a similar comment to `remote_span` would help here.

Reply:
Could you please precise? I don't understand what you want to be changed.

Reply:
I was proposing to add a comment similar to the one in `remote_span` https://github.com/oneapi-src/oneDPL/blob/distributed-ranges/include/oneapi/dpl/internal/distributed_ranges_impl/shp/remote_span.hpp#L14-L17 clarifying that the only change over a regular vector is the rank.

Reply:
ok, applied